docs/reference/commandline/image_prune.md

```diff
@@ -82,6 +82,7 @@ which removes images with the specified labels. The other
 format is the `label!=...` (`label!=<key>` or `label!=<key>=<value>`), which removes
 images without the specified labels.
 
+> [!NOTE]
 > **Predicting what will be removed**
 >
 > If you are using positive filtering (testing for the existence of a label or
```

Comment:
Does the GitHub syntax still allow for a custom heading? For some of these, the `NOTE` is now taking the place of the heading, and a bit boiler plating without much added value.

<img width="359" alt="Screenshot 2024-08-19 at 16 19 31" src="https://github.com/user-attachments/assets/65a0704d-5c23-4be2-8af8-3480ba84f7fd">

Reply:
They don't allow custom headings. The alert types are hard-coded to: 

```
note
tip
important
warning
caution
```